<ui version="4.0" >
 <class>modCalcEquinox</class>
 <widget class="QWidget" name="modCalcEquinox" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Equinoxes and Solstices</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>485</width>
        <height>474</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Interactive Mode</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Equinoxes and solstices for the year:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KIntNumInput" name="Year" />
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QGridLayout" >
           <item row="0" column="0" >
            <widget class="QLabel" name="VEquinoxText" >
             <property name="text" >
              <string>Vernal equinox:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QLabel" name="VEquinox" >
             <property name="font" >
              <font>
               <family>Monospace</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text" >
              <string>1 Jan 2007  00:00</string>
             </property>
             <property name="indent" >
              <number>5</number>
             </property>
             <property name="textInteractionFlags" >
              <set>Qt::TextSelectableByMouse</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QLabel" name="SSolsticeText" >
             <property name="text" >
              <string>Summer solstice:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" >
            <widget class="QLabel" name="SSolstice" >
             <property name="font" >
              <font>
               <family>Monospace</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text" >
              <string>1 Jan 2007  00:00</string>
             </property>
             <property name="indent" >
              <number>5</number>
             </property>
             <property name="textInteractionFlags" >
              <set>Qt::TextSelectableByMouse</set>
             </property>
            </widget>
           </item>
           <item row="2" column="0" >
            <widget class="QLabel" name="AEquinoxText" >
             <property name="text" >
              <string>Autumnal equinox:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" >
            <widget class="QLabel" name="AEquinox" >
             <property name="font" >
              <font>
               <family>Monospace</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text" >
              <string>1 Jan 2007  00:00</string>
             </property>
             <property name="indent" >
              <number>5</number>
             </property>
             <property name="textInteractionFlags" >
              <set>Qt::TextSelectableByMouse</set>
             </property>
            </widget>
           </item>
           <item row="3" column="0" >
            <widget class="QLabel" name="WSolsticeText" >
             <property name="text" >
              <string>Winter solstice:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1" >
            <widget class="QLabel" name="WSolstice" >
             <property name="font" >
              <font>
               <family>Monospace</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text" >
              <string>1 Jan 2007  00:00</string>
             </property>
             <property name="indent" >
              <number>5</number>
             </property>
             <property name="textInteractionFlags" >
              <set>Qt::TextSelectableByMouse</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="eqPlotWidget" name="Plot" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>485</width>
        <height>474</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Batch Mode</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="HelpLabel" >
         <property name="text" >
          <string>Specify years for the calculation in the input file.</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" >
         <item row="0" column="0" >
          <widget class="QLabel" name="label_3" >
           <property name="text" >
            <string>Input file:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="KUrlRequester" name="InputFileBatch" />
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_4" >
           <property name="text" >
            <string>Output file:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="KUrlRequester" name="OutputFileBatch" />
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" >
           <item>
            <widget class="QPushButton" name="RunButtonBatch" >
             <property name="text" >
              <string>Compute</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ViewButtonBatch" >
             <property name="text" >
              <string>View output</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>465</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>QPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>eqPlotWidget</class>
   <extends>KPlotWidget</extends>
   <header>eqplotwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
